app/helpers/rad_common/application_helper.rb

```diff
@@ -188,6 +201,18 @@ def export_button(model_name)
               class: 'btn btn-secondary btn-sm')
     end
 
+    def show_help_menu?(user)
+      !user_signed_in? || user.external?
+    end
+
+    def onboarded?
```

Comment:
do we need this? can this usage be moved inside the view partial [here](https://github.com/cannapages/cannasaver_admin/blob/main/app/views/layouts/_navigation.html.haml#L10)?

Reply:
@elbzero  do we need this? can this usage be moved inside the view partial [here](https://github.com/cannapages/cannasaver_admin/blob/main/app/views/layouts/_navigation.html.haml#L10)?